kubernetes/yamls/development/elasticsearch/es-rc.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: v1
+kind: ReplicationController
```

Comment:
Make it a deployment instead of replication controller. (Deployment is the newer and better version of a replication container) 
https://kubernetes.io/docs/concepts/workloads/controllers/deployment/